core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -586,8 +587,8 @@ private TempBlockMeta createBlockMetaInternal(long sessionId, long blockId,
       if (newBlock) {
         checkTempBlockIdAvailable(blockId);
       }
-      StorageDirView dirView =
-          mAllocator.allocateBlockWithView(sessionId, initialBlockSize, location, getUpdatedView());
+      StorageDirView dirView = mAllocator.allocateBlockWithView(sessionId,
+          initialBlockSize, location, new StorageMetadataView(mMetaManager));
```

Comment:
nit; did you consider calling `StorageMetadatView` `BlockMetadataManagerView` instead and renaming the existing `BlockMetadataManagerView` to `BlockMetadataManagerEvictableView` for consistency?